DUI3-DX.slnf

```diff
@@ -25,6 +29,7 @@
       "DUI3-DX\\Sdk\\Speckle.Connectors.Utils\\Speckle.Connectors.Utils.csproj",
       "DUI3-DX\\Sdk\\Speckle.Converters.Common.DependencyInjection\\Speckle.Converters.Common.DependencyInjection.csproj",
       "DUI3-DX\\Sdk\\Speckle.Converters.Common\\Speckle.Converters.Common.csproj",
+      "DesktopUI3\\DUI3\\DUI3.csproj",
```

Comment:
This is the old DUI3 project, and shoulden't be included in the DX slnf

Reply:
I'll take this @JR-Morgan and we can I can add it to a list of outstanding issues

Reply:
@clairekuang I think the warnings might need resolving @clairekuang before we can pull it in actually